### DISCUSSION TITLE
Function based view support?

### DISCUSSION THREAD

**Discussion by jdbit** [abhiabhi94/dj-hitcount#34] [Q&A] [answered]
Hi,
do you have any example how to use dj-hitcount with a function based view?

**Comment by abhiabhi94** [comment, marked as answer]
Hey @jdbit, thanks for the question. 

Does this help: https://dj-hitcount.readthedocs.io/en/latest/installation.html#hitcountmixin. Basically, you can directly call the method `hit_count` from your view function, since it is a `staticmethod`. Let me know, in case you have any more doubts.

**Reply by jdbit** [reply]
Thank you! I'll try it.
